tests/samples_tests/smoke_tests/common/samples_common_test_class.py

```diff
@@ -80,7 +81,12 @@ def prepend(cache, inp='', model=''):
     if inp:
         inp = '-i', download(test_data_dir, test_data_dir / inp)
     if model:
-        model = '-m', download(test_data_dir, test_data_dir / model)
+        if type(model) is ov.ie_api.Model:
+            model_sv_path = test_data_dir / "tmp.xml"
+            ov.save_model(model, model_sv_path)
+            model = '-m', model_sv_path
```

Comment:
Please, save the model to `tmp_path` instead of `test_data_dir`. `test_data_dir` is a  read only cache because the tests can be run in multiple threads, `tmp_path` is unique for every test and automatically deleted by pytest after 3 runs.

Reply:
Done